github_actions/README.md

```diff
@@ -0,0 +1,154 @@
+# GitHub Action for running pylint commands
+
+Each time that new code is pushed into your repo, you can have a [pylint](https://docs.pylint.org) command automatically run.
+
+Example workflow:
+* Put the following text into a file named `.github/main.workflow` in your repo):
+```hcl
+workflow "on push" {
+  on = "push"
+  resolves = ["GitHub Action for pylint"]
+}
+
+action "GitHub Action for pylint" {
+  uses = "PyCQA/pylint/github_actions@master"
+  # args = "pip install -r requirements.txt ; pylint **/*.py"
+  args = "pip install . ; pylint **/*.py"
+}
+```
+Or to add other pylint options to __args =__ above.
+
+$ __pylint -h__
```

Comment:
I'm just curious, would this `pylint -h` output get automatically updated every time the upstream `pylint` gets updated?

Reply:
Yeah, maybe we don't need the whole pylint -h output, and we can just link to the documentation ?